Makefile

```diff
@@ -1,33 +1,46 @@
-all: clean dist
+all: tests clean dist
 
+.PHONY: install_deps # install requirements for the test suite
+install_deps:
+	pip install tox
+
+.PHONY: tests  # run tox test suite
+tests: install_deps
+	tox
```

Comment:
ideally, we should steer clear of (ab)using Make to automate tasks, and stick to using `setup.py`, which is a common idiom when working with Python projects

Reply:
ok, but I don't think there is a way to run tox from setup.py in our case, is there? I wanted the test procedure to be relatively independent from the CI (just run `make tests` regardless of the environment, CI, local dev machine...)

Reply:
we could add a `make tests` wrapper for convenience, dependencies should be installed via `setup.py` (functional) or `requirements/[...].txt`